docs/Administrator Guide/Directory Quota.md

```diff
@@ -203,14 +203,22 @@ on client side.
 
 Use the following command to update the memory cache size:
 
+1. Soft Timeout : The frequency at which the quota server-side translator checks the volume usage when the usage is below the soft limit. The soft timeout is in effect when the disk usage is less than the soft limit.
+
+```console
+# gluster volume set VOLNAME features.soft-timeout time
```

Comment:
Can we change it to `# gluster volume set <VOLNAME> features.soft-timeout <time>` which is followed in other places like [1].

The only reason of asking this would this is it's much more clear to the user where to put values in the command and makes the doc easy to understand.

Feel free to different if you don't like the idea.
 

[1] https://github.com/gluster/glusterdocs/blob/master/docs/Administrator%20Guide/Mandatory%20Locks.md

Reply:
Ack.